dispatches/case_studies/renewables_case/tests/test_RE_flowsheet.py

```diff
@@ -121,34 +120,46 @@ def test_create_model_PV():
 
 def test_wind_battery_optimize(input_params):
```

Comment:
Please add a few sentence comment explaining the system being simulated for this test. For example, is this a X MW wind + Y MW, Z MWh battery being simulated for a week? What grid modeling assumptions were made?

Reply:
A comment link this will help a new user work backwards to understand the settings that went into running the test. While the code likely already specifies all of the settings, a new user might not have the big picture of what is going on. The comment I am suggesting to add provides the big picture.